code-samples/creating-your-bot/event-handling/.gitignore

```diff
@@ -0,0 +1 @@
+config.json
```

Comment:
Would be good renaming `config.json` to `config.json.sample` to be used as a template for the people using this code base, if file changes no more tracked -git-ignored-

Reply:
We can make the change